### PR TITLE
Qualys API Security Plugin

### PR DESCRIPTION
**Pull request by pmgupte**
# Description

Github repository: https://github.com/jenkinsci/qualys-api-security-plugin
Original repository: https://github.com/Qualys/qualys-api-security-plugin
Hosting request is https://issues.jenkins-ci.org/projects/HOSTING/issues/HOSTING-831
Committers: @qualys-plugins @pmgupte @lseletos @gwaghmare @pranaybqualys @nikita-bedmutha @gouripatilqualys

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

#### Merge permission to GitHub repository
- [x] Check this if newly added person also needs to be given merge permission to the GitHub repo.

### CODE REVIEW

File: permissions/plugin-qualys-api-security.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "qualys-api-security"
+github: "jenkinsci/qualys-api-security-plugin"
+paths:
+	- "com/qualys/plugins/qualys-api-security-plugin"
```
Comment (slide) [inline permissions/plugin-qualys-api-security.yml]: Looks like you have invalid whitespace in your yaml.

Comment (pmgupte) [reply permissions/plugin-qualys-api-security.yml]: There are tabs, which, I guess need to be removed.

Comment (pmgupte) [reply permissions/plugin-qualys-api-security.yml]: Tabs are removed, and a build has passed after that.

File: permissions/plugin-qualys-api-security.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "qualys-api-security"
+github: "jenkinsci/qualys-api-security-plugin"
+paths:
+- "com/qualys/plugins/qualys-api-security-plugin"
```
Comment (slide) [inline permissions/plugin-qualys-api-security.yml]: The `paths` item is your pom's `groupId` with periods replaced with `/` plus your `artifactId`, which in this case is `qualys-api-security`. So, you will need to remove `-plugin` from your path.

Comment (pmgupte) [reply permissions/plugin-qualys-api-security.yml]: Thanks for pointing that out @slide ! Appreciate your attention to details.

Comment (pmgupte) [reply permissions/plugin-qualys-api-security.yml]: removed `-plugin` from paths.

**Comment by qualys-plugins** [comment]
@slide The Jenkins community user `qualys` mentioned in this YAML file belongs to github user `qualys-plugins`. However, the github user `qualys-plugins` hasn't got write permissions on the Github repository: https://github.com/jenkinsci/qualys-api-security-plugin

is there anything pending for this?

**Comment by slide** [comment]
Please try now.

**Comment by pmgupte** [comment]
@slide works now. Thanks @jenkinsadmin

**Pull request merged**